pkgs/by-name/gi/github-mcp-server/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+  versionCheckHook,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "github-mcp-server";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "github";
+    repo = "github-mcp-server";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-LpD4zLAeLFod7sCNvBW8u9Wk0lL75OmlRXZqpQsQMOs=";
+  };
+
+  vendorHash = "sha256-YqjcPP4elzdwEVvYUcFBoPYWlFzeT+q2+pxNzgj1X0Q=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X=main.version=${finalAttrs.version}"
+    "-X=main.commit=${finalAttrs.src.rev}"
+    "-X=main.date=1970-01-01T00:00:00Z"
+  ];
+
+  doInstallCheck = true;
+  nativeInstallCheckInputs = [ versionCheckHook ];
+  versionCheckProgramArg = "--version";
+
+  meta = {
+    description = "GitHub's official MCP Server";
+    homepage = "https://github.com/github/github-mcp-server";
```

Comment:
I just noticed since Github publishes a changelog, we should probably have that
```
changelog = "https://github.com/github/github-mcp-server/releases/tag/v${finalAttrs.version}";
```

Reply:
Fixed, thanks!